.github/workflows/build-multiarch.yml

```
@@ -27,35 +27,40 @@ jobs:
           - java11-openj9
         include:
           - variant: java17
+            baseImage: eclipse-temurin:17-jre
+            platforms: linux/amd64,linux/arm/v7,linux/arm64
+            mcVersion: LATEST
+          - variant: java17jdk
```

These need to be dash delimited, like
```suggestion
          - variant: java17-jdk
```